test/unit/errors/error-collector.test.js

```diff
@@ -667,6 +743,17 @@ tap.test('Errors', (t) => {
         t.notHas(params, 'stack_trace')
         t.end()
       })
+
+      t.test('should have a transaction id', (t) => {
+        const transactionId = errorJSON[5]
+        t.not(transactionId, undefined)
```

Comment:
same suggestion from above, can you assert what the actual value is since you know it?

Reply:
I do have access to the transaction so it's now fixed as recommended.